src/main/java/orangehrm/OrangeRecruitmentPage.java

```diff
@@ -0,0 +1,246 @@
+package orangehrm;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.List;
+
+public class OrangeRecruitmentPage {
+    private final WebDriver driver;
+    private WebElement createdVacancy;
+    private WebElement createdCandidate;
+    private String title;
+    private String recordVacancyName;
+
+    @FindBy(id = "menu_recruitment_viewJobVacancy")
+    private WebElement vacancies;
+
+    @FindBy(id = "btnAdd")
+    private WebElement addButton;
+
+    @FindBy(id = "addJobVacancy_jobTitle")
+    private WebElement jobTitleList;
+
+    @FindBy(id = "addJobVacancy_name")
+    private WebElement vacancyName;
+
+    @FindBy(id = "addJobVacancy_hiringManager")
+    private WebElement hiringManager;
+
+    @FindBy(id = "addJobVacancy_noOfPositions")
+    private WebElement numberOfPositions;
+
+    @FindBy(id = "addJobVacancy_description")
+    private WebElement description;
+
+    @FindBy(id = "addJobVacancy_status")
+    private WebElement status;
+
+    @FindBy(id = "addJobVacancy_publishedInFeed")
+    private WebElement feedCheckbox;
+
+    @FindBy(id = "btnSave")
+    private WebElement saveBtn;
+
+    @FindBy(id = "btnBack")
+    private WebElement backBtn;
+
+    @FindBy(xpath = "//table[@id='resultTable']//tr[@class='odd' or @class='even']")
+    private List<WebElement> resultList;
+
+    @FindBy(id = "menu_recruitment_viewCandidates")
+    private WebElement candidates;
+
+    @FindBy(id = "addCandidate_firstName")
+    private WebElement firstName;
+
+    @FindBy(id = "addCandidate_lastName")
+    private WebElement lastName;
+
+    @FindBy(id = "addCandidate_email")
+    private WebElement email;
+
+    @FindBy(id = "addCandidate_contactNo")
+    private WebElement contactNo;
+
+    @FindBy(id = "addCandidate_vacancy")
+    private WebElement candidateVacanciesList;
+
+    @FindBy(id = "addCandidate_comment")
+    private WebElement candidateComment;
+
+    @FindBy(id = "addCandidate_appliedDate")
+    private WebElement appliedDate;
+
+    @FindBy(id = "addCandidate_consentToKeepData")
+    private WebElement consentDataCheckbox;
+
+    public OrangeRecruitmentPage(WebDriver driver){
+        PageFactory.initElements(driver, this);
+        this.driver = driver;
+    }
+
+    public WebElement getCreatedVacancy() {
+        return createdVacancy;
+    }
+
+    public WebElement getCreatedCandidate() {
+        return createdCandidate;
+    }
+
```

Comment:
As already mentioned, it is not good idea to have getter for WebElement variable.

Reply:
fixed

Reply:
I still see getters for webelements